app/scripts/components/facets/facets.controllers.ts

```diff
@@ -98,6 +99,10 @@ module ngApp.components.facets.controllers {
       this.FacetService.addTerm(facet, term);
     }
 
+    addMissing(facet: string): void {
+      this.FacetService.addTerm(facet, 'MISSING');
```

Comment:
api will be updated to allow `vital_status in [blood, _missing]` as well as `vital_status = blood or vital_status is missing`